src/main/java/com/shigotosagashi/jobdomainservice/controller/JobController.java

```diff
@@ -0,0 +1,49 @@
+package com.shigotosagashi.jobdomainservice.controller;
+
+import com.shigotosagashi.jobdomainservice.dto.out.ExampleEntityDto;
+import com.shigotosagashi.jobdomainservice.repository.ExampleEntityRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.persistence.EntityNotFoundException;
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/job")
+public class JobController {
+
+  @Autowired
+  private ExampleEntityRepository exampleEntityRepository;
+
+  @PostMapping("/save")
+  public ExampleEntityDto save(
+      @Valid @RequestBody com.shigotosagashi.jobdomainservice.dto.in.ExampleEntityDto example) {
```

Comment:
Qual o benefício de ter a importação dessa classe em cada declaração de método, no lugar de manter centralizada na seção de imports?

Reply:
I've added two kinds of DTOs, one for receive (in) a request body, and another for responding (out) a request.
Both of them have the same name, that's why I had to full import this one here.
I can rename them, we just need to agree upon a pattern.

Reply:
I've created DTOs with different suffix. 'In' to 'dto.in' and 'Out' to 'dto.out'.